http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/FootballTeamResource.java

```diff
@@ -0,0 +1,80 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import org.jboss.logging.Logger;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.smallrye.mutiny.Uni;
+
+/**
+ *
+ * When a JSON extension is installed such as quarkus-rest-jackson or quarkus-rest-jsonb
+ * Quarkus will use the application/json media type by default for most return values
```

Comment:
This comment is true https://quarkus.io/guides/rest#declaring-endpoints-representation-content-types however I can't see relation to this resource as there is just one endpoint that consumes/produces something else than HTTP status (no entity), that is `uploadFootballJson` and there, you explicitly delcare these annotations.

So, why to state what is default if you override it and don't test it?